packages/sbcl/termux-tweaks.patch

```diff
@@ -0,0 +1,69 @@
+diff --git a/contrib/sb-posix/posix-tests.lisp b/contrib/sb-posix/posix-tests.lisp
+index 7c94a43..c565fad 100644
+--- a/contrib/sb-posix/posix-tests.lisp
++++ b/contrib/sb-posix/posix-tests.lisp
+@@ -527,7 +527,7 @@
+   3 #\f #\o #\o)
+ 
+ (deftest opendir.1
+-  (let ((dir (sb-posix:opendir "/")))
++  (let ((dir (sb-posix:opendir "/data/data/com.termux/cache")))
```

Comment:
Don't use this app cache: use[ our local tmp directory instead](https://github.com/termux/termux-packages/blob/master/packages/slang/slang-2.3.2_slsh_lib_rline_editor.sl.patch#L9), same below.

Reply:
Thanks.

Reply:
Works. In one place the leading `/` had to be removed from `TERMUX_PREFIX`.